tools/clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -780,8 +783,14 @@ void clang::EmitBackendOutput(DiagnosticsEngine &Diags,
     // so that future passes can be skipped.
     AsmHelper.EmitAssembly(Action, OS);
   } catch (const ::hlsl::Exception &hlslException) {
+#if !defined(DXC_CODEGEN_EXCEPTIONS_ABORT)
     Diags.Report(Diags.getCustomDiagID(DiagnosticsEngine::Error, "%0\n"))
         << StringRef(hlslException.what());
+#else
+    fprintf(stdout, "internal error: %s\n", hlslException.what());
```

Comment:
This should probably be writing to `llvm::errs()` rather than to `stdout`.

Reply:
It turns out llvm::errs() doesn't do anything in release builds on Linux in libdxcompiler.so,
nor when the dxc command line compiler traps here.

I filed #6774 to followup.

I do want the output in those cases though.